smriprep/workflows/surfaces.py

```diff
@@ -355,16 +377,20 @@ def _dedup(in_list):
         return vals.pop()
 
     workflow.connect([
-        (inputnode, autorecon3, [('use_T2', 'use_T2'),
-                                 ('use_FLAIR', 'use_FLAIR')]),
+        (inputnode, autorecon_surfs, [('use_T2', 'use_T2'),
+                                      ('use_FLAIR', 'use_FLAIR')]),
```

Comment:
Do you think this PR will help with #33 in any way? Is there any dataset with FLAIR images on OpenNeuro that we can use to test?

Reply:
No, there are no changes to how FLAIRs are handled.